qutebrowser/browser/adblock.py

```diff
@@ -234,16 +228,13 @@ def _parse_line(self, line):
         parts = line.split()
         if len(parts) == 1:
             # "one host per line" format
-            host = parts[0]
-        elif len(parts) == 2:
-            # /etc/hosts format
```

Comment:
That comment probably still applies to the `else:` part?

Reply:
oops yes, readded